docs/setup/install/deb.asciidoc

```diff
@@ -22,7 +22,7 @@ include::key.asciidoc[]
 
 [source,sh]
 -------------------------
-wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
+wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo gpg --dearmor -o /usr/share/keyrings/elasticsearch-keyring.gpg && sudo chmod 0644 /usr/share/keyrings/elasticsearch-keyring.gpg
```

Comment:
I'm unsure if changing permissions is needed. #127645 proposed that change, but was closed without merging. @nik9000, since you approved the original Elasticsearch PR for this change, do you have opinions here?

Reply:
You should be able to test it with the same commands I ran before. I'll give it a go locally. One moment!

Reply:
I was able to install kibana without having to do the chmod. But 0644 is the permission I got by default when running dearmor.

Reply:
I just tested on a VM running Debian 11 and did not have to modify permissions.